zlib.h.in

```diff
@@ -57,11 +57,11 @@ extern "C" {
 #define ZLIBNG_VER_STATUS 0         /* 0=devel, 1-E=beta, F=Release */
 #define ZLIBNG_VER_MODIFIED 0       /* non-zero if modified externally from zlib-ng */
 
-#define ZLIB_VERSION "1.2.11.zlib-ng"
-#define ZLIB_VERNUM 0x12bf
+#define ZLIB_VERSION "1.2.12.zlib-ng"
+#define ZLIB_VERNUM 0x12cf
 #define ZLIB_VER_MAJOR 1
 #define ZLIB_VER_MINOR 2
-#define ZLIB_VER_REVISION 11
+#define ZLIB_VER_REVISION 12
 #define ZLIB_VER_SUBREVISION 0
```

Comment:
I think perhaps SUBREVISION should also be F, it surprised me a little to see we didn't have that set already. Would have to check up on that, but still not really something for this PR.

Reply:
Because it's decimal, it can't be `F`... Of course it could be 15, which is decimal equivalent of 0xF...